src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Separete these imports  to improve maintainability and readability of the codebase.

https://levelup.gitconnected.com/how-to-sort-imports-in-react-project-550f5ce70cbf

```
import React from 'react';

import { Person } from './components/Person/Person';

import './App.scss';
```